test/soak/serve_hostnames/serve_hostnames.go

```diff
@@ -40,24 +40,25 @@ import (
 )
 
 var (
-	queries = flag.Int("queries", 100, "Number of hostname queries to make in each iteration")
-	perNode = flag.Int("per_node", 1, "Number of hostname pods per node")
-	upTo    = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
+	queriesAverage = flag.Int("queries", 10, "Number of hostname queries to make in each iteration per pod on average")
+	podsPerNode    = flag.Int("pods_per_node", 1, "Number of serve_hostname pods per node")
+	upTo           = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
+	verbose        = flag.Bool("verbose", false, "Report extra details about operations")
 )
 
 const (
 	deleteTimeout        = 2 * time.Minute
 	endpointTimeout      = 5 * time.Minute
 	podCreateTimeout     = 2 * time.Minute
-	podStartTimeout      = 5 * time.Minute
+	podStartTimeout      = 10 * time.Minute
```

Comment:
Have you really seen it take 10 minutes for the pods to be running (isn't this is a tiny image)?